src/main/java/seedu/address/logic/parser/EditStudentCommandParser.java

```diff
@@ -39,29 +39,30 @@ public EditCommand parse(String args) throws ParseException {
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
         } catch (ParseException pe) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditStudentCommand.MESSAGE_USAGE), pe);
         }
 
-        EditPersonDescriptor editPersonDescriptor = new EditPersonDescriptor();
+        EditStudentDescriptor editStudentDescriptor = new EditStudentCommand.EditStudentDescriptor();
```

Comment:
shouldn't have reference to `EditStudentCommand`, should just be `EditStudentDescriptor `since is has already been imported